.travis.yml

```diff
@@ -1 +1,23 @@
+sudo: false
+
+os:
+ - linux
+ - osx
```

Comment:
If you want the os x testers to actually run, you have to email travis-ci and ask them to enable them manually.